packages/core/strapi/lib/services/server/middleware.js

```diff
@@ -45,7 +53,7 @@ const resolveMiddlewares = (config, strapi) => {
 
       middlewares.push({
         name: item,
-        handler: middlewareFactory({}, { strapi }),
+        handler: instanciateMiddleware(middlewareFactory, item, {}, strapi),
       });
```

Comment:
Should be `instantiateMiddleware` rather than `instanciateMiddleware`

Reply:
Thank you, fixed!